swan-lake/integration-tutorials/sending-emails-from-a-service.md

```diff
@@ -612,85 +478,42 @@ function getEmailContent(int appointmentNumber, Appointment appointment, Payment
 
 ### Step 3: Build and run the service
 
-You can run this service by navigating to the project root and using the `bal run` command.
+![Run the service](/learn/images/integration-tutorials/sending-emails-from-a-service/run_the_service.gif)
```

Comment:
At least for others let's follow /Users/username/integration-tutorials/tutorial-name as the root.

Reply:
+1